src/defaults.js

```diff
@@ -34,6 +77,92 @@ const DEFAULT_SKYDB_OPTIONS = {
   portalFileFieldname: "file",
 };
 
+/**
+ * Removes a prefix from the beginning of the string.
+ *
+ * @param str - The string to process.
+ * @param prefix - The prefix to remove.
+ * @param [limit] - Maximum amount of times to trim. No limit by default.
+ * @returns - The processed string.
+ */
+const trimPrefix = function (str, prefix, limit) {
```

Comment:
Can we put all the utilities in a different file? This file should only have code related to defaults.

Reply:
Yes, I agree.
I first put everything into defaults.js because I didn't know where to insert which functions.

Everything in "src/utils.js" or new files?

Reply:
`utils.js` is fine, or you can follow the file structure in `skynet-js` since we're picking up more utils.